callstats-java-sdk/src/main/java/io/callstats/sdk/CallStatsHttpClient.java

```diff
@@ -55,6 +55,16 @@
 	/** The so time out. */
 	private int soTimeOut;
```

Comment:
This is auto generated with some plugin, has to be re written once before release.